### PR TITLE
jquery1

### PR DESCRIPTION
**Pull request by krystianan**

### CODE REVIEW

File: jquery1/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
+﻿<html>
```
Comment (matwjablonski) [inline jquery1/index.html]: add doctype

Comment (matwjablonski) [reply jquery1/index.html]: and lang attribute

File: jquery1/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    <head>
+        <meta charset-"utf-8">
+        <title>Js6</title>
+        <link rel="stylesheet" href="css/style.css">
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+        </head>
```
Comment (matwjablonski) [inline jquery1/index.html]: remove one tab

File: jquery1/js/scripts.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+$(function(){
+    console.log('DOM załadowany - można się bawić');
+});
+
+var span = $('span');
```
Comment (matwjablonski) [inline jquery1/js/scripts.js]: this and below code should be in your function which load dom document

**Review by matwjablonski — CHANGES_REQUESTED** [review]

**Comment by krystianan** [comment]
change index
delete 'console.log('DOM załadowany - można się bawić');' because it was unnecessary

File: jquery1/js/scripts.js [hunk 4]
```diff
@@ -1,7 +1,3 @@
-$(function(){
```
Comment (matwjablonski) [inline jquery1/js/scripts.js]: why you delete it?

**Review by matwjablonski — CHANGES_REQUESTED** [review]

File: jquery1/js/scripts.js (hunk 4 above)
Comment (krystianan) [reply jquery1/js/scripts.js]: it was copied from other file, not function that was needed in this task

**Pull request merged**